packages/validate/test/Validator.test.js

```diff
@@ -1,9 +1,22 @@
 import { expect, fixture, html, unsafeStatic, defineCE } from '@open-wc/testing';
-import { LitElement } from '@lion/core';
 import sinon from 'sinon';
+import { LitElement } from '../../core/index.js';
```

Comment:
that needs to stay `@lion/core` otherwise it only works within the monorepo and not for our users 🙈

Reply:
Ofcourse, new commit pushed 😊

Reply:
tests are not real user code - still, from an npm package point of view it is an "external" package so let's be consistent 👍

Reply:
nice 👌

Reply:
Yes ofcourse, i agree. I updated it because i first did "npm i" instead of "yarn install" 😊